src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -175,26 +369,33 @@ func startCluster(
 		return err
 	}
 
-	reqPorts := n * 5 // we need 5 ports per node
-	ports, err := vnet.GetFreePortPool(int(reqPorts))
-	if err != nil {
-		return err
-	}
-
 	// Start a seed node.
 	var (
 		seedID            uint
-		seedKafkaPort     = ports[0]
-		seedProxyPort     = ports[1]
-		seedSchemaRegPort = ports[2]
-		seedRPCPort       = ports[3]
-		seedMetricsPort   = ports[4]
+		seedKafkaPort     = cPorts.KafkaPorts[0]
+		seedProxyPort     *uint
+		seedSchemaRegPort *uint
+		seedRPCPort       *uint
+		seedMetricsPort   *uint
```

Comment:
[question] Why are you using pointers for the ports?

Reply:
hmm it is technically valid to use port 0 but I think standards prevent it so let me change that instead.